src/main/frontend/modules/outliner/pipeline.cljs

```diff
@@ -41,14 +41,36 @@
                                            [(:db/id (:block/page block))]
                                            (map :db/id (:block/refs block))
                                            parents-refs))
+                            ;; Usually has changed as new-refs has page id while old-refs doesn't
                             refs-changed? (not= old-refs new-refs)
```

Comment:
I don't know if it's intentional but this was always changed when I inspected it because the old-refs didn't contain the page id

Reply:
It's a bug because `:block/path-refs` is included in the [retracted-attributes](https://github.com/logseq/logseq/blob/master/deps/db/src/logseq/db/schema.cljs#L105), the `block` here is from the `db-after` which has no path-refs already. I made the [change](https://github.com/logseq/logseq/pull/9254/commits/a63d4e408f5b2b1c32e0d90b09f27027f5d4d61d) to get the old-refs from `db-before`.